wireapp/kalium
# feat: join existing MLS conversations on login AR-2060

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

Implements: [Use case: join existing conversation from a new device (MLS)](https://wearezeta.atlassian.net/wiki/spaces/ENGINEERIN/pages/576454924/Use+case+join+existing+conversation+from+a+new+device+MLS)

During the slow sync we send a request to join all existing MLS conversations.

#### Todo:
- [x] Execute join requests in parallel

### Testing

#### Test Coverage (Optional)


#### How to Test

1. Login to an account which has previously been added to MLS conversations
2. Check the logs that we are sending request to join these conversations3. 
### Notes (Optional)

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
